StudentAdvisementSystem/web/WEB-INF/web.xml

```diff
@@ -21,7 +21,8 @@
         </session-timeout>
     </session-config>
     <welcome-file-list>
-        <welcome-file>faces/studentFolder/profile.xhtml</welcome-file>
+        <!--<welcome-file>faces/studentFolder/profile.xhtml</welcome-file>-->
```

Comment:
I just commented out the student welcome page so I could load the advisors page until we can get both to work.